apps/src/applab/designElements/label.jsx

```diff
@@ -264,14 +366,17 @@ export default {
         })
         .appendTo($(document.body));
 
-      const padding = parseInt(element.style.padding, 10);
+      const {
+        horizontalPadding,
+        verticalPadding
+      } = elementUtils.calculatePadding(element.style.padding);
 
       if (!widthLocked) {
         // Truncate the width before it runs off the edge of the screen
-        size.width = Math.min(clone.width() + 1 + 2 * padding, maxWidth);
+        size.width = Math.min(clone.width() + 1 + horizontalPadding, maxWidth);
```

Comment:
Why +1?

Reply:
Good question, I didn't change this part of the logic.

Reply:
👍